samples/simple/QuickConsoleTest/Program.cs

```diff
@@ -9,8 +9,9 @@
 var builder = Host.CreateDefaultBuilder(args)
     .ConfigureServices(services =>
     {
-        var home = Path.Join(Environment.CurrentDirectory, "..", "..", "..", "..", "ExamplePythonDependency");

```

Comment:
This path resolve line is not working while running in a GitHub Codespace.
Best